### ISSUE TITLE
A11y_Visual Studio Code Services_Blog Posts_Extension Bisect_Keyboard: Keyboard tab focus moves to non-interactive element "IN THIS BLOG POST" 

### ISSUE THREAD

**Issue by prabhudevu** [microsoft/vscode-docs#5540]
“[Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

**GitHubTags**:#A11yUsable;#A11yUsablehigh;#A11yTCS;#BM_Visual Studio Code Services_Aug2022;#Visual Studio Code Services;#Win11;#FTP;#DesktopWeb;#A11ySev4;#Chrome;#Keyboard;

## Environment Details:
Application Name: Visual Studio Code Services
URL: https://code.visualstudio.com/
Windows Version: Win11
Chrome Version: Version 103.0.5060.134 (Official Build) (64-bit)

## Repro Steps:

1. Please do not close this bug. This bug should only be closed by TCS, C+AI Accessibility tester after verification of the fix
2. Launch the application Url : https://code.visualstudio.com/
3. Tab Navigate to "Blog" link.
4. Tab Navigate to Right Navigation 
5. Observe that Keyboard focus is going to non-interactive element "IN THIS BLOG POST"

## Actual:
Keyboard tab focus moves to non-interactive element "IN THIS BLOG POST" 

## Expected:
Non-interactive elements should not receive keyboard focus on tab navigation

## User Impact:
Keyboard users will think it is an interactive control and try to perform an action on it on doing so they would fail

## Attachment
![5540_A11y_Visual Studio Code Services_Blog Posts_Extension Bisect](https://user-images.githubusercontent.com/98821792/183020291-15d87c06-c8ad-4c1a-beac-46726ed30bde.gif)

**Comment by gregvanl** [comment]
Fixed and closing as no "Resolved" state for GitHub issues
